poko-compiler-plugin/src/main/kotlin/dev/drewhamilton/poko/ir/equalsGeneration.kt

```diff
@@ -214,6 +215,7 @@ private fun IrBuilderWithScope.irCallContentDeepEquals(
  * function if it represents a primitive array.
  */
 context(IrPluginContext)
+@OptIn(UnsafeDuringIrConstructionAPI::class)
```

Comment:
```suggestion
@OptIn(UnsafeDuringIrConstructionAPI::class)
context(IrPluginContext)
```
In my brain the context is part of the signature

Reply:
Makes sense - I just tried though and there's a compiler error:

![image](https://github.com/user-attachments/assets/fab03a8f-f2ef-4048-ade8-6c9da3236e11)

Reply:
Let this awkward syntax be a reminder to remove context receivers from this project 😬